ten-simple-rules-dockerfiles.Rmd

```diff
@@ -526,7 +530,7 @@ Unless specific features are needed, bind mounts are also preferable to [storage
 Standalone _script files_ are distinct from other software as they are treated as files and managed software, i.e. installed with a package manager and versioned.
 If you developed extensive software for a specific analysis, you should ideally publish it as a software package and follow \ruleref{rule:pinning} for installing it (cf. "A package first" in @gruening_recommendations_2019).
 Consider using the suitable package system with pinned versions even for small scripts if the functions are reusable across datasets or workflows by you or others.
-You should avoid installing software packages from source after `COPY`ing the code into the image, because the connection between the file outside of the image and the one copied in is easily lost (cf.&nbsp;\ruleref{rule:mount}).
+You should avoid installing software packages from source after `COPY`ing the code into the image, because the connection between the file outside of the image and the one copied in is easily lost.
```

Comment:
I know this is in the original paper, but it reads a little funny. I understand the point to copy files AFTER install, but I don't understand the comment about a connection being lost. Can anyone comment on this?

Reply:
That's right, I didn't actually edit this (it might just be a change in whitespace due to my editor) so I'll wait for the original author to comment. I did tweak the surrounding sentences for clarity though.

Reply:
I've run into that too - if someone doesn't knit it looks like a new change. If you didn't edit this, then it's out of scope for this PR and we can look at it later.

Reply:
The point I was trying to make is that when you copy files in and install from there, you don't know the source of the files... _which does not make sense_, because the `COPY` statement mentions the source clearly. And the next sentence makes source that the source files are managed in the same repo with the Dockerfile. I'll update this in my next PR.

Reply:
Changed in https://github.com/nuest/ten-simple-rules-dockerfiles/commit/390f259e41cc5658383bec39e7c9aebf0a52a8f1 and included in https://github.com/nuest/ten-simple-rules-dockerfiles/pull/68